spec/integration/user_index_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+describe 'User Index Page Features', type: :feature, js: true do
+  before :each do
+    @user1 = User.create(
+      name: 'Harley Quinn',
+      photo: 'https://images.unsplash.com/photo-1696537768609-1cf03f53e893?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=1374&q=80',
+      bio: 'user1 Pix',
+      posts_counter: 5
+    )
+  end
+  # I can see the username of all other users.
+  it 'can see the username of all other users' do
+    visit users_path
+    expect(page).to have_all_of_selectors('.user-info')
+  end
+  # I can see the profile picture for each user.
+  it 'can see the profile pic of each users' do
+    visit users_path
```

Comment:
- [ ] Great work adding the missing tests as requested by the previous reviewer but all the integration tests are failing. Please make sure that all tests pass without any error.

![image](https://github.com/rabayarabu/RoR-Blog-App/assets/83803414/88678fda-1e58-49b4-b78a-7016798941c4)